### PR TITLE
Remove reference to deprecated command in man page

### PR DESCRIPTION
**Pull request by Loquacity**
Issue: https://github.com/SUSE/spacewalk/issues/5977

Per BZ#1109526
See also doc update: https://github.com/SUSE/doc-susemanager/pull/226

### CODE REVIEW

**Comment by mantel** [comment]
I think the version number should be bumped so we know for sure which version we are using. Lara, for some strange reason there are two places that need to get updated: src/smdba/smdba and setup.py . Just search for "VERSION" or "version", you cannot miss it. Currently master is at 1.6.2

File: setup.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 setup (
   name = 'SUSE Manager Database Control',
-  version = '1.6.2',
+  version = '1.6.3',
```
Comment (isbm) [inline setup.py]: Could you please update the version?

**Comment by isbm** [comment]
Conflicts here. I think because of another PR was already merged prior this. Could you please resolve the conflicts and update the version?

**Comment by mantel** [comment]
> Conflicts here. I think because of another PR was already merged prior this. Could you please resolve the conflicts and update the version?

Yes, I bumped the version number with my latest merge. When doing this I also noticed that the versions were inconsistent: The package version did not match what the tool itself printed. Reason is that the version number lives in two places and they need to match.

See here for details:

https://github.com/SUSE/smdba/pull/28/files

**Comment by mantel** [comment]
Ok, I resolved the conflict. Hope this is ok, as I was unsure whether this request was directed to Lana or to me :/

**Comment by Loquacity** [comment]
Do you need anything from me here?

**Comment by isbm** [comment]
@Loquacity sorry for being too slow here, won't happen again. You've made changes against the branch that is now called "1.6". Could you please rebase on it? We are working on a port to Python3 only and all further branches won't work on Py2 versions.

**Comment by Loquacity** [comment]
> @Loquacity sorry for being too slow here, won't happen again. You've made changes against the branch that is now called "1.6". Could you please rebase on it? We are working on a port to Python3 only and all further branches won't work on Py2 versions.

So the original PR  was to remove the db-check command from the man page. I just checkout out the 1.6 branch, and it is not documented in that version of the man page, so there is nothing to do there. Feel free to idiot-check me, but I think this can probably be closed now without merging.

**Comment by juliogonzalez** [comment]
>  Feel free to idiot-check me, but I think this can probably be closed now without merging.

@isbm, can this be closed then?

**Pull request closed without merging**